### DISCUSSION TITLE
Code missing?

### DISCUSSION THREAD

**Discussion by dineshbvadhia** [NRWLDev/fastapi-problem#16] [Q&A] [answered]
https://github.com/NRWLDev/fastapi-problem/blob/main/src/fastapi_problem/handler/fastapi.py

Is it just on my machine or has the code disappeared?

**Comment by EdgyEdgemond** [comment, marked as answer]
https://github.com/NRWLDev/fastapi-problem/blob/main/src%2Ffastapi_problem%2Fhandler%2F__init__.py

the code is moving here, i haven't added a deprecation warning yet. So its reimported to its old location, imports should continue to work. 

in future it will be `from fastapi_problem.handler import ...` rather than `from fastapi_problem.handler.xxx import ...`

**Reply by EdgyEdgemond** [reply]
https://github.com/NRWLDev/fastapi-problem/issues/15

Flagged for update

**Reply by EdgyEdgemond** [reply]
https://github.com/NRWLDev/fastapi-problem/releases/tag/v0.7.17
